.github/workflows/release.yml

```
@@ -36,3 +36,14 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NODE_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
+
+      - name: setup Node to publish to GitHub Packages
+        uses: actions/setup-node@v1
+        with:
+          registry-url: 'https://npm.pkg.github.com'
+          node-version: '12.x'
+
+      - name: publish to GitHub Packages
+        run: npm publish
```

This project is used `Yarn`
```suggestion
        run: yarn publish
```